src/components/VActivity.vue

```diff
@@ -40,6 +41,7 @@ const isFiltered = useIsFiltered(props.activity)
 			hasRead && $style.read,
 			!isFiltered && $style.hidden,
 		]"
+		v-show="!hideRead || (hideRead && !hasRead)"
```

Comment:
I think it'd be better to implement this logic as another filter so that the bizarre style logic for showing/hiding borders continues to work.

`v-show` isn't used because `$style.hidden` adds the `display: none` for when an activity is filtered out.

Reply:
Cool, I'm going to draft this until you've looked into that. I'm curious how you'll abstract filters beyond the current single priority filter.